### PR TITLE
Create funktionstjanster.se.xml

### PR DESCRIPTION
**Pull request by hstseverywhere**
Related: #17767

### CODE REVIEW

**Comment by hstseverywhere** [comment]
Related: #17767

File: src/chrome/content/rules/funktionstjanster.se.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		*.me.funktionstjanster.se
+		*.testidp.funktionstjanster.se
+
+	See also: 
+
+		- https://doc.funktionstjanster.se/
+-->
+
+<ruleset name="funktionstjanster.se">
+	<target host="*.idp.funktionstjanster.se" />
```
Comment (J0WI) [inline src/chrome/content/rules/funktionstjanster.se.xml]:
```
doc.funktionstjanster.se
eformular.funktionstjanster.se
etp.funktionstjanster.se
mefiles.funktionstjanster.se
```

File: src/chrome/content/rules/funktionstjanster.se.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+		*.idp.funktionstjanster.se  (most traffic is on this one)
+
+		*.idpat.funktionstjanster.se
+		*.idpst.funktionstjanster.se
+		*.me.funktionstjanster.se
+		*.testidp.funktionstjanster.se
+
+	See also: 
+
+		- https://doc.funktionstjanster.se/
```
Comment (J0WI) [inline src/chrome/content/rules/funktionstjanster.se.xml]: Non-working:
```
https://eidt.funktionstjanster.se:18898/
https://grpt.funktionstjanster.se:18898/
```

File: src/chrome/content/rules/funktionstjanster.se.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<!-- CGI eID-tjänst: authenticators and formmails for .SE gov sites etc
+
+	known authenticator subdomains:
+
+		*.idp.funktionstjanster.se  (most traffic is on this one)
+
+		*.idpat.funktionstjanster.se
+		*.idpst.funktionstjanster.se
+		*.me.funktionstjanster.se
+		*.testidp.funktionstjanster.se
```
Comment (J0WI) [inline src/chrome/content/rules/funktionstjanster.se.xml]: Can they be added as targets?

**Comment by pipboy96** [comment]
I will try to fix up this ruleset.

**Comment by ivysrono** [comment]
@pipboy96 There are only 3 old ruleset PRs : this, #14807 #14438 .
Maybe we can done them these days?
I will be busy after 3 days.

**Comment by pipboy96** [comment]
@ivysrono Do you want me to try and fix up these now?

**Comment by ivysrono** [comment]
Tomorrow or the day after tomorrow.

**Comment by ivysrono** [comment]
@pipboy96 em, I have requested a review from J0WI 8 hours ago. Let's wait at least 1 day?

**Comment by pipboy96** [comment]
@ivysrono Yes, I think we may wait.

**Pull request merged**